session/config.go

```diff
@@ -29,3 +31,10 @@ type Config struct {
 	ProfilePathMemory string
 	ProfileHTTPAddr   string
 }
+
+// FlowConfig is a Flow config.
+type FlowConfig struct {
```

Comment:
And let's move this to the `flow` package and rename it to `flow.Config`.

Reply:
Per offline comment, will keep the `FlowConfig` in `session` to avoid import cycle.